src/main/resources/templates/swift/xcode/readme

```diff
@@ -0,0 +1,44 @@
+# Introduction to SwiftUI
+
+In this exercise, we want to implement a Hello World SwiftUI example via Xcode.
```

Comment:
Shouldn't it be:
```suggestion
In this exercise, we want to implement a Hello Artemis SwiftUI example via Xcode.
```
?

Reply:
I would argue that every programming language has a "Hello World" example, instead of e.g. "Hello Java", etc. 
That is why. I wouldn't mind to change it though.

Reply:
Yes, sure, but since the "exercise" is to change it to "Hello Artemis", I thought about changing it. What about combining both?

Reply:
Good point, I'll use Hello Artemis :)

Reply:
👍🏼